k2/csrc/array_ops_test.cu

```diff
@@ -1663,4 +1663,15 @@ TEST(OpsTest, Array2Assign) {
   }
 }
 
+TEST(OpsTest, SizesToMergeMapTest) {
+  for (int loop = 0; loop < 10; loop++) {
```

Comment:
Seems below test is only a simple testing, we don't need to `loop 10` here. I guess you plan to add random tests here. 
Anyway, at least we can add a TODO (for random case, as kenerls definitely need to be tested with random case)

Reply:
will reduce to 2